test/commons/dom/has-content-virtual.js

```diff
@@ -1,7 +1,7 @@
 /* global xit */
-describe('dom.hasContent', function () {
+describe('dom.hasContentVirtual', function () {
   'use strict';
-  var hasContent = axe.commons.dom.hasContent;
+  var hasContentVirtual = axe.commons.dom.hasContentVirtual;
```

Comment:
In the test file for `test/commons/aria/label.js`, we're testing `label` directly. But here, we're testing `has-content-virtual`. Should we test both, or make the tests more consistent?